src/zarr/api/asynchronous.py

```diff
@@ -765,9 +765,9 @@ async def open_group(
 
 
 async def create(
-    shape: ChunkCoords,
+    shape: ChunkCoords | int,
     *,  # Note: this is a change from v2
-    chunks: ChunkCoords | None = None,  # TODO: v2 allowed chunks=True
+    chunks: ChunkCoords | int | None = None,  # TODO: v2 allowed chunks=True
```

Comment:
we must not have test coverage here but you could also throw in `bool` and remove this TODO.